views/vt_ksm_phs.sql

```diff
@@ -0,0 +1,57 @@
+create or replace view vt_ksm_phs as
+
+With KSM_PHS AS (Select comm.id_number,

```

Comment:
Can you use the committee_phs table view here instead of the committee table directly?

Select *
From table(rpt_pbh634.ksm_pkg.tbl_committee_phs)